frontend/openchat-client/src/openchat.ts

```diff
@@ -482,6 +482,10 @@ export class OpenChat extends EventTarget {
         if (this._identity === undefined) {
             throw new Error("onCreatedUser called before the user's identity has been established");
         }
+        if (user.suspended) {
+            alert("User account suspended");
```

Comment:
alert???

Reply:
MVP

Reply:
And they don't even get alerted in their own language?